### DISCUSSION TITLE
"get schema" doesnt work anymore with flutter flow

### DISCUSSION THREAD

**Discussion by SalTheSWE** [supabase/supabase#21661] [Questions]
i have the free plan on supabase and back when i started creating the schema in supabase i was able to easily port it onto flutter flow but now for some reason it doesnt work, yes the database has grown bigger but is this because i have the free plan and i have passed the limit on something?

**Comment by dshukertjr** [comment]
No, the plan shouldn't matter and you should still be able to run the `Get Schema` from your FlutterFlow account. Does it give you any specific error message or anything when you press the `Get Schema` button?

**Comment by taamorim** [comment]
I'm having same error. No error message. :-(
